lighty-netconf-device/src/main/java/io/lighty/netconf/device/requests/RpcOutputRequestProcessor.java

```diff
@@ -58,6 +60,13 @@ public RpcDefinition getRpcDefinition() {
         return rpcDefinition;
     }
 
+    public Absolute getRpcDefInputAbsolutePath() {
+        final ArrayList<QName> qnames = new ArrayList<>();
+        qnames.add(this.rpcDefinition.getQName());
+        qnames.add(this.rpcDefinition.getInput().getQName());
+        return Absolute.of(qnames);
```

Comment:
Please replace with: return Absolute.of(rpcDefinition.getQName(), rpcDefinition.getInput().getQName());

Reply:
Done